azure-pipelines.yml

```diff
@@ -188,7 +191,6 @@ jobs:
   #         else
   #           echo "Skipping Danger JS because API token wasn't found"
   #         fi
-
```

Comment:
I still don't understand how this is running formatting for you when it never does for me. My changes to these files never get formatted on commit. Is it an IDE plugin for you? (This will cause a bunch of unnecessary conflicts too, I'm removing this section of code entirely in another PR.)

Reply:
I don't know why prettier ignore file doesn't have azure pipeline yml files. But that's why - the IDE plugin will format everything not listed in prettier ignore. It should have done so for you as well... if you use that plugin :P

Reply:
In my change yesterday, I actually just added yml to the list of file types that will be formatted. I can revert that if it's not desirable, but I don't really see any reason why we shouldn't be formatting them.

Reply:
I think we should format them. I think that will promote consistency.